### DISCUSSION TITLE
Submit feature requests here!

### DISCUSSION THREAD

**Discussion by Vagr9K** [Vagr9K/gatsby-advanced-starter#103] [Ideas]
You can submit your feature requests here!

**Comment by elbuendiscipulo** [comment]
- Search/filter function for the blog - filter on tags, categories, search on blog post titles.
- Change text size in blog post, + - buttons that allows the user to change text size on the blog posts, useful for mobile readers.

**Comment by elbuendiscipulo** [comment]
- Table of contents for blog posts - with links to each section.  
Heres a good example: 
https://firstsiteguide.com/what-is-blog-post/ 
<img width="1114" alt="Screen Shot 2022-01-02 at 18 05 34" src="https://user-images.githubusercontent.com/94419591/147892795-cae0f1f9-796c-48c6-ae1c-eed87e68a3ad.png">

**Comment by elbuendiscipulo** [comment]
Add meta-keyword to articles: 
https://www.screamingfrog.co.uk/seo-spider/user-guide/tabs/#meta-keyword

I see that the file `/themes/advanced/src/components/SEO/RichSearch.tsx` uses the tags as keywords. 
Is this something similar? 

Still the SEO crawler Screaming Frog says my blog posts has no meta-keywords.

**Comment by elbuendiscipulo** [comment]
Add Title tag to blog posts. 

Now the blog posts use the heading as title tag. 

https://www.siteguru.co/seo-academy/heading-vs-title-tags
